migrations/sqls/20190324132907-dbdev-up.sql

```diff
@@ -0,0 +1,182 @@
+/* Replace with your SQL commands */
+
+/*Create person.person table */
+CREATE TABLE IF NOT EXISTS person (
+  personId int AUTO_INCREMENT PRIMARY KEY,
+  personType nvarchar(32) NOT NULL, /*PersonType (volunteer,boardmember,client,donor) */
+  firstName nvarchar(255) NOT NULL,
+  lastName nvarchar(255) NOT NULL,
+  dob date,
+  gender nvarchar(32) NOT NULL,  /*Gender constraint at front end */
+  dateAdded DATETIME DEFAULT CURRENT_TIMESTAMP,
+  dateModified DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  addedBy int NOT NULL /*Self Reference to Person ID*/,
+  modifiedBy int NOT NULL /*Self Reference to Person ID*/,
+  FOREIGN KEY (addedBy) REFERENCES person(personId),
+  FOREIGN KEY (modifiedBy) REFERENCES person(personId)
+);
+/*drop recreate command*/
+#DROP TABLE person;
+/*Crate person Contanct table-Contact M:1 Person*/
+CREATE TABLE IF NOT EXISTS contact (
+  personId int NOT NULL, /*Foreign key reference to person table */
+  contactId int AUTO_INCREMENT PRIMARY KEY,
+  primaryContact varchar(32),
+  primaryCarrier varchar(32),
+  textMessages int, /*Would client like to recieve text messages from CU*/
+  emergencyContact varchar(32),
+  email varchar(128),
+  dateAdded DATETIME DEFAULT CURRENT_TIMESTAMP,
+  addedBy int NOT NULL, /*Foreign key reference to person table */
+  FOREIGN KEY (personId) REFERENCES person(personId),
+  FOREIGN KEY (addedBy) REFERENCES person(personId)
+);
+#DROP TAbLE contact;
+/*Create Demographics Table M:1 Person*/
+CREATE TABLE IF NOT EXISTS demographics (
+  personId int NOT NULL, /*foreing key reference to person table*/
+  demographicId int AUTO_INCREMENT PRIMARY KEY,
+  originCountry varchar(64),
```

Comment:
It might be better to store the two or three Country Code instead of the full name. The country code is considered more definitive and resilient to changes to a country's name.